src/models/Settings.js

```diff
@@ -49,6 +49,7 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
       'stateToken': ['string', false, undefined],
       'username' : ['string', false],
       'signOutLink': ['string', false],
+      'isSamlForceAuthn': ['boolean', true, false],
```

Comment:
Do we need to add a new variable? I think we can get away with an existing one, ```features.rememberMe```. If  you pass that ```false``` the widget should hide the checkbox for remember me. 
The reason I am suggesting that is because it seems that this new variable is only used for the same purpose.

Reply:
true, i can just check for the feature in PrimaryAuth.js